### PR TITLE
Fix serialization issue with W&B

### PR DESCRIPTION
**Pull request by jjzhuo**
The chain input_documents are not displaying properly in W&B, due to serialization issue:

Screenshot 2023-06-04 at 11 58 26 AM

### CODE REVIEW

**Comment by hwchase17** [comment]
@parambharat how does this look to you?

File: langchain/callbacks/tracers/wandb.py [hunk 1]
```diff
@@ -79,7 +85,7 @@ def _convert_chain_run_to_wb_span(trace_tree: Any, run: Run) -> trace_tree.Span:
 
 def _convert_tool_run_to_wb_span(trace_tree: Any, run: Run) -> trace_tree.Span:
     base_span = _convert_run_to_wb_span(trace_tree, run)
-    base_span.results = [trace_tree.Result(inputs=run.inputs, outputs=run.outputs)]
+    base_span.results = [trace_tree.Result(inputs=serialize_inputs(run.inputs), outputs=run.outputs)]
```
Comment (parambharat) [inline langchain/callbacks/tracers/wandb.py:88]: @jjzhuo this should be `_serialized_inputs`

**Comment by parambharat** [comment]
> The chain input_documents are not displaying properly in W&B, due to serialization issue:
> 
> Screenshot 2023-06-04 at 11 58 26 AM

Hi @jjzhuo , can you share an example of where you are seeing this ? A colab perhaps ?

**Review by parambharat — CHANGES_REQUESTED** [review]

**Comment by jjzhuo** [comment]
@parambharat The screenshot is from the [Weights and Biases](https://wandb.ai/) website.

You can reproduce it by adding a RetrievalQA in this notebook:
https://python.langchain.com/en/latest/integrations/agent_with_wandb_tracing.html

**Comment by parambharat** [comment]
@jjzhuo , Thanks you for the fix. I have verified that the new `_serialize_inputs` method serializes the inputs of the `StuffDocumentsChain` in the following [wandb run](https://wandb.ai/parambharat/langchain-tracing/runs/sflt59a2?workspace=user-parambharat). 

@hwchase17 , This LGTM.

**Review by parambharat — APPROVED** [review]
LGTM

**Review by hwchase17 — APPROVED** [review]
thanks!

**Pull request merged**